README.md

```diff
@@ -38,40 +39,26 @@ a multi-system chiptune tracker.
   - ability to change tempo mid-song with `Cxxx` effect (`xxx` between `000` and `3ff`)
 - open-source under GPLv2 or later.
 
-## downloads
+***
+# quick references
 
-check out the [Releases](https://github.com/tildearrow/furnace/releases) page. available for Windows, macOS and Linux (AppImage).
-
-## discussion
-
-see the [Discussions](https://github.com/tildearrow/furnace/discussions) section, or (preferably) the [official Discord server](https://discord.gg/EfrwT2wq7z).
-
-## help
-
-check out the [documentation](papers/doc/README.md). it's mostly incomplete, but has details on effects.
-
-# unofficial packages
+ - **downloads**: check out the [Releases](https://github.com/tildearrow/furnace/releases) page. available for Windows, macOS and Linux (AppImage).
```

Comment:
This makes "downloads" less notorious (which is probably the first thing a new user seeks).

Reply:
I've also moved the downloads listing to be a subheading, near the top of the page.